FILES.deploy-tools/root/group_vars/loadbal.yml

```diff
@@ -1,3 +1,3 @@
 ---
 k8_initial_labels: "ndslabs-node-role=loadbalancer"
-security_groups: "default,remote HTTP,HTTPS"
+security_groups: "k8s_loadbalancer"
```

Comment:
As far as I can tell, this is still the cause of the load-balancer failure to start up.  If I revert your security group changes, things work as expected.